gapis/api/vulkan/draw_call_mesh.go

```diff
@@ -293,7 +293,7 @@ func getVerticesData(ctx context.Context, s *api.State, thread uint64,
 	fullSize := uint64(vertexCount-1)*stride + perVertexSize
 
 	offset := uint64(attribute.Offset) + (uint64(firstVertex) * stride)
-	if offset > vertexSlice.count || offset+fullSize > vertexSlice.count {
+	if offset >= vertexSlice.count || offset+fullSize > vertexSlice.count {
```

Comment:
The `=` in the LHS should only matter if the `fullSize` is `0`. I don't think it can be `0`, but adding `=` just make it more correct. (Although I did observed a crash once which escaped the original guard, and this quite odd to me too)

I think the range to be used for slice is [offset, offset + fullSize). So I think RHS shouldn't have '='.